Perlang.ConsoleApp/Perlang.ConsoleApp.csproj

```diff
@@ -3,6 +3,8 @@
     <PropertyGroup>
         <OutputType>Exe</OutputType>
         <TargetFramework>netcoreapp3.1</TargetFramework>
+        <PublishTrimmed>true</PublishTrimmed>
```

Comment:
This uses the Mono Linker to make the output smaller. More info: https://github.com/mono/linker/blob/master/src/ILLink.Tasks/README.md

It would also be useful to look into some of the suggestions given here, which can make us build a more convenient "fat" executable. But we need to ensure that we don't loose too much in runtime performance by doing so. (in fact, the R2R suggestions given there are likely to _improve_ startup performance) https://www.hanselman.com/blog/MakingATinyNETCore30EntirelySelfcontainedSingleExecutable.aspx